playground/examples/vue/full/toolbar.vue

```diff
@@ -115,6 +115,15 @@ const editor = useEditor<EditorExtension>({ update: true })
       <div :class="Themes.ICON_H3" />
     </Button>
 
+    <Button
+      :pressed="editor.nodes.horizontalRule.isActive()"
+      :disabled="!editor.commands.insertHorizontalRule.canExec()"
+      tooltip="Horizontal Rule"
```

Comment:
Let's use the term "Divider". "Horizontal Rule" is a bit technical for most people.